DynamicSmoothedGUI.py

```diff
@@ -291,11 +343,13 @@ def updateGridSmoothed(self):
         try:
             if self.desired_heading is not None and self.heading == self.desired_heading:
                 self.draw_headings()
+                self.drawC1C0()
                 self.output_state = "move forward"
                 print(self.output_state)
 
             if self.desired_heading is not None and self.heading != self.desired_heading:
                 self.draw_headings()
+                self.drawC1C0()
```

Comment:
Remove, drawC1C0 is already called from the first if block, that executes no matter what

Reply:
do you mean remove from line 352? i can't do that because i need that there to show the turning process and update the arrow.

Reply:
Ya ur correct it needs to stay 😄